lib/src/base-styles/_nx-text-input.scss

```diff
@@ -6,32 +6,52 @@
  */
 
 .nx-text-input {
-  border: $nx-border-dark;
-  border-left-width: 4px;
-  border-radius: 0;
-  box-shadow: none;
-  height: $nx-form-element-height;
-  padding: 0 8px;
   width: $nx-form-element-width-normal;
+}
+
+.nx-text-input__box {
+  align-items: center;
+  border: 1px solid #d1d6f0;
+  border-radius: $nx-border-radius;
+  display: flex;
+}
+
+.nx-text-input__input {
+  background-color: transparent;
+  border: none;
+  color: $nx-text-color-light;
+  flex-grow: 1;
+  font-size: $nx-font-size;
+  line-height: $nx-line-height;
+  padding: $nx-spacing-xs $nx-spacing-md;
```

Comment:
I feel like this padding should probably be on the box; with the input itself having none

Reply:
> I feel like this padding should probably be on the box; with the input itself having none

The commit diff looks weird: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/be4cce51f3c368f60b3e7089c9fbb628121170eb